src/ripple/test/jtx/impl/Env.cpp

```diff
@@ -60,8 +60,11 @@ Env::AppBundle::AppBundle(Application* app_)
 Env::AppBundle::AppBundle(std::unique_ptr<Config const> config)
 {
     auto logs = std::make_unique<Logs>();
-    owned = make_Application(
-        std::move(config), std::move(logs));
+    auto timeKeeper_ =
+        std::make_unique<ManualTimeKeeper>();
+    timeKeeper = timeKeeper_.get();
```

Comment:
Note to other reviewers: `timeKeeper` isn't referenced anywhere but here now, but there are future plans for it. eg. https://github.com/vinniefalco/rippled/commit/75532cb449716536c129e12e9cf2ad843be48837#diff-1fa292ad88b698cfe1b304837d40f7f9R124